contributing-guides/style-guide.md

```diff
@@ -104,40 +104,74 @@ Example:
 Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 ```
 
-## Token syntax
+## Placeholder syntax
 
-User-provided values should use the `{{token}}` syntax
+User-provided values should use the `{{placeholder}}` syntax
 in order to allow `tldr` clients to highlight them.
 
-Keep the following guidelines in mind when choosing tokens:
+### General rules
 
-### Naming
-- Use short but descriptive tokens,
-  such as `{{source_file}}` or `{{wallet.txt}}`.
-- Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word tokens.
-- Use an actual value rather than a generic placeholder where appropriate.
-  For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+- Use short but descriptive placeholders,
+  such as `{{path/to/source.cs}}` or `{{drive}}`.
+- Use [snake_case][snake_case] for multi-word placeholders.
+- Use an actual value in placeholder such as `{{5}}` rather than
+  a placeholder with a value description inside it such as `{{seconds}}`
+  when one of the following conditions are met:
+  - there is a long (consisting of several letters) option immidiately before
+    this placeholder 
+  - there is a short (consisting of one letter) option immidiately before this
+    placeholder with a mnemonic provided via square brackets in a code
+    example description
+  - the command or it's the most nested subcommand performs one simple task such as `sleep` 
+   
+  For example, use `iostat {{2}}` rather than `iostat {{seconds}}`.
+
+> :bulb: More formal rules are described below.
+
+[snake_case]: https://wikipedia.org/wiki/snake_case
+
+### Quoting
+
+- Don't quote path placeholders for page simplicity although shell may require quotes.
+  In other cases quote placeholders just when:
+  - documentation mandates quotes explicitly
+  - you exactly know they are required and the documentation just missing information about this
+- Use double quotes for placeholders where variable or another substitution occures.
+  In other cases use single quotes.
```

Comment:
This is overly specific. Let's remove this item from the list.

Reply:
I can move all formal rules to a separate file and simplify this style guide. From my perspective it's a bad idea to remove this rule because it let us to know when and what prefer exactly. :)